examples/assets/attributions.md

```diff
@@ -51,7 +51,7 @@ Originally in HDR format
 
 ## spruit_sunrise_1k
 
-spruit_sunrise_1k.hdr, spruit_sunrise_1k.jpg by <a href="https://hdrihaven.com">HDRI Haven</a>
+spruit_sunrise_1k.hdr, spruit_sunrise_1k.jpg, spruit_sunrise_2k_HDR.hdr, spruit_sunrise_2k_LDR.jpg by <a href="https://hdrihaven.com">HDRI Haven</a>
```

Comment:
I switched us from 2k environments to 1k, since that matches our 256 cubemaps. I think adding these was just a merge conflict, since I was trying to rename them.

Reply:
Will fix